examples/nextjs-with-typescript/pages/index.tsx

```diff
@@ -2,68 +2,19 @@ import Link from "next/link";
 
 function HomePage() {
   return (
-    <div>
-      <h1>Browse Mux Elements</h1>
+    <nav>
       <ul>
-        <li>
-          <h3>
-            <Link href="/MuxVideo">
-              <a>Mux Video Demo</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/MuxAudio">
-              <a>Mux Audio Demo</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/MuxPlayer">
-              <a>Mux Player Demo</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/MuxPlayerDynamic">
-              <a>Mux Player Demo (Dynamically Loaded)</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/MuxPlayerIframe">
-              <a>Mux Player Demo (Loaded inside an iframe with fullscreen disabled)</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/MuxUploader">
-              <a>Mux Uploader with Mux Uploader Drop Demo</a>
-            </Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/mux-video">Mux Video (Web Component) Demo</Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/mux-audio">Mux Audio (Web Component) Demo</Link>
-          </h3>
-        </li>
-        <li>
-          <h3>
-            <Link href="/mux-player">Mux Player (Web Component) Demo</Link>
-          </h3>
-        </li>
+        <li><Link href="/MuxVideo"><a className="video">&lt;MuxVideo&gt;</a></Link></li>
+        <li><Link href="/MuxAudio"><a className="audio">&lt;MuxAudio&gt;</a></Link></li>
+        <li><Link href="/MuxPlayer"><a className="player">&lt;MuxPlayer&gt;</a></Link></li>
+        <li><Link href="/MuxPlayerDynamic"><a className="player">&lt;MuxPlayer&gt; (dynamic)</a></Link></li>
+        <li><Link href="/MuxPlayerIframe"><a className="player">&lt;MuxPlayer&gt; (w/o fullscreen)</a></Link></li>
```

Comment:
Maybe worth mentioning it's using an iframe explicitly?
```suggestion
        <li><Link href="/MuxPlayerIframe"><a className="player">&lt;MuxPlayer&gt; (<iframe> w/o fullscreen)</a></Link></li>
```

Reply:
I tried this but it's too long for the box :( it'll require 2 lines messing up the layout a bit

Reply:
`(iframe w/o fs)`?